src/controller.ts

```diff
@@ -48,7 +48,7 @@ export class AugurNodeController {
   }
 
   public async start(errorCallback: ErrorCallback | undefined) {
-    // startFetchingGasPrice(); removing b/c of runtime issue in AWS
+    startFetchingGasPrice();
```

Comment:
Move this call to after block stream processing starts, no need to get gas prices during bulk sync because the websocket server isnt even started so augur-node wont be accepting requests.